pkg/app/src/components/Account/modules/campaigns-section/contributed-campaigns-section.tsx

```diff
@@ -0,0 +1,137 @@
+import React, { FC } from 'react'
+import {
+	Box,
+	Card,
+	CardContent,
+	CardMedia,
+	LinearProgress,
+	Table,
+	TableBody,
+	TableCell,
+	TableHead,
+	TableRow,
+	Typography,
+	Chip,
+	TableContainer,
+} from '@mui/material'
+import { tempCampaginContributions } from '../../TempData/index'
+import { useTheme } from '@mui/material/styles'
+
+interface ContributionStatus {
+	active: String
+	completed: String
+}
+
+const status: ContributionStatus = {
+	active: 'Active',
+	completed: 'Completed',
+}
+
+const ContributedCampaginsSection: FC = () => {
+	const theme = useTheme()
+	console.log(theme.palette.common.black)
```

Comment:
please do not use console.log, use a dedicated logger, so we can globally control if and where we want to log.
@DarkNebula0 @developer-sidani @hadi-ibra

Reply:
Noted

Reply:
can you give an example of other loggers